packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/simulate-ensemble-util.ts

```diff
@@ -30,3 +42,62 @@ export function formatSimulateModelConfigurations(
 
 	return [...Object.values(ensembleModelConfigMap)];
 }
+
+export function getChartEnsembleMapping(
+	node: WorkflowNode<SimulateEnsembleCiemssOperationState>,
+	stateToModelConfigMap: { [key: string]: string[] }
+) {
+	const wfOutputState = getActiveOutput(node)?.state;
+	const mapping = _.clone(wfOutputState?.mapping ?? []);
+
+	// For every State Variable that has not been mapped in the ensembleMapping
+	// We will fill in here so the user can still see these if they want
```

Comment:
The comment is a bit confusing - what are you trying to say?

Reply:
This is trying to say that we will add individual model options for the user to select on top of the ensemble mapping that was provided for the run.

The following picture the user has only mapped `Sus` but the user should still be allowed to select charts for `Sus`, `S`, `I` and `R` 
<img width="413" alt="image" src="https://github.com/user-attachments/assets/150882ad-9591-44f8-9073-d4cbe9053c1b" />